src/Authoring/DiagnosticTests/PositiveData.cs

```diff
@@ -1,10 +1,67 @@
-namespace DiagnosticTests
-{
-    public partial class TestDiagnostics
+namespace DiagnosticTests
+{
+    public partial class UnitTesting
+    {
+        private const string Valid_ListUsage = @"
+using System.Collections.Generic;
+namespace Test 
+{
+    public sealed class A
+    {
+        public static IReadOnlyList<Uri> GetUris()
+        {
+            return new List<Uri>() {
+                new Uri(""http://github.com""),
+                new Uri(""http://microsoft.com"")
+                };
+        }
+    }
+}";
+
+ private const string Valid_ListUsage2 = @"
+namespace Test 
+{
+    public sealed class A
+    {
+        public static System.Collections.Generic.IReadOnlyList<Uri> GetUris()
+        {
+            return new System.Collections.Generic.List<Uri>() {
+                new Uri(""http://github.com""),
+                new Uri(""http://microsoft.com"")
+                };
+        }
+    }
+}";
+
+
+        // Namespaces
+        private const string Valid_NestedNamespace = @"
+// Assuming the winmd is RuntimeComponent, the error gets thrown since namespace `A` has no common prefix with RuntimeComponent

```

Comment:
Isn't this a positive test case file?

Reply:
Yes this is a valid test, the comment is a copy-paste error, fixed in last commit